execution/result/suiteResult.go

```diff
@@ -40,6 +40,8 @@ type SuiteResult struct {
 	ProjectName       string
 	Timestamp         string
 	SpecsSkippedCount int
+	PreHookMessage    []string
```

Comment:
Should be `PreHookMessages` and `PostHookMessages` (plural), since this is an array. This is particularly important in the contract since proto would generate methods like `GetPreHookMessage` which indicates a singular value.